packages/story-editor/src/elements/media/imageDisplay.js

```diff
@@ -83,7 +83,12 @@ function ImageDisplay({ element, box, previewMode }) {
     if (resourceList.get(resource.id)?.type !== 'fullsize' && resource.src) {
       timeout = setTimeout(async () => {
         const url = getProxiedUrl(resource, resource.src);
-        const preloadedImg = await preloadImage(url, srcSet);
+        let preloadedImg;
+        try {
+          preloadedImg = await preloadImage(url, srcSet);
+        } catch (error) {
+          //ignore
```

Comment:
There should be a `return` here.

If you just `catch`, accessing `preloadedImg.currentSrc` below will cause an error because `preloadedImg` is `undefined`.

Also, can just do `catch {`, without `erorr`.